pkgs/applications/networking/instant-messengers/jitsi-meet-electron/default.nix

```diff
@@ -0,0 +1,65 @@
+{ stdenv
+, lib
+, fetchurl
+, makeWrapper
+, electron_8
+, dpkg
+, gtk3
+, glib
+, gsettings-desktop-schemas
```

Comment:
you're going to need to add such dependencies to `buildInputs` for `wrapGAppsHook` to work

Reply:
actually, now that I think of it `electron` should be wrapped and creating a wrapper with wrapGAppsHook should be uneeded (the electron binary should be wrapped the same way afterall)

It seems it's being wrapped in the wrong phase https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/tools/electron/generic.nix#L61

Reply:
https://github.com/NixOS/nixpkgs/issues/84913